package.json

```diff
@@ -46,6 +45,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.8.2",
     "jest": "^23.1.0",
+    "mocha": "^5.2.0",
     "webpack": "^4.11.1",
     "webpack-cli": "^2.1.4"
   }
```

Comment:
Clean out dependencies that you no longer need, or move them to dev dependencies (faker, new relic, nodemon)